src/components/Pagination/pageButtons.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import getFirstItem from './get-first-item';
+
+export default function PageButtons(props) {
+    const { pages, activePage, onChange } = props;
+
+    const getButtonClassName = (firstItem, index, buttonsToRender) => (
+        classnames(
+            'rainbow-pagination_button',
+            {
+                'rainbow-pagination_button--first': index === 0,
+                'rainbow-pagination_button--last': index === buttonsToRender - 1,
+                'rainbow-pagination_button--active': activePage === firstItem + index,
+            },
+        )
+    );
+
+    const getAriaCurrent = (page) => {
+        if (page === activePage) {
+            return 'page';
+        }
+        return undefined;
+    };
+
+    const renderButtons = () => {
+        const firstItem = pages > 4 ? getFirstItem(pages, activePage) : 1;
+        const buttonsToRender = pages > 4 ? 5 : pages;
+        return Array(buttonsToRender).fill(0).map((item, index) => {
+            const page = firstItem + index;
+            return (
+                <li key={`page-button-${page}`} className={getButtonClassName(firstItem, index, buttonsToRender)}>
```

Comment:
since inside getButtonClassName you use ```firstItem + index``` then pass the constant page directly here